### DISCUSSION TITLE
SVG Images

### DISCUSSION THREAD

**Discussion by jag-sandhu** [Azure/CCOInsights#255] [Q&A] [answered]
Hi All,
Am I the only one effected where svg images are not displayed in the dashboard. i.e. I get a 404 when browsing to
https://azure.github.io/ccodashboard/asets/2020_Icons/SecurityCenter.svg

whereas it the report should be using:
https://github.com/Azure/CCOInsights/tree/master/docs/assets/2020_Icons
?

Thanks,

**Comment by josunefon** [comment, marked as answer]
Hi @jag-sandhu 

Problem fixed with the new templates, the reason was because we changed the name of the repo from ccodashboard to ccoinsights.
